libraries/AP_Notify/Buzzer.h

```diff
@@ -38,22 +36,19 @@ class Buzzer: public NotifyDevice
     /// on - turns the buzzer on or off
     void on(bool on_off);
 
-    // Patterns - how many beeps will be played
-    enum BuzzerPattern {
-        NONE = 0,
-        SINGLE_BUZZ = 1,
-        DOUBLE_BUZZ = 2,
-        GPS_GLITCH = 3, // not used
-        ARMING_BUZZ = 4,
-        BARO_GLITCH = 5,
-        EKF_BAD = 6
-    };
-
-    /// play_pattern - plays the defined buzzer pattern
-    void play_pattern(BuzzerPattern pattern_id);
+    // Patterns - how many beeps will be played; read from
+    // left-to-right, each bit represents 100ms
+    static const uint32_t    SINGLE_BUZZ = 0b10000000000000000000000000000000UL;
+    static const uint32_t    DOUBLE_BUZZ = 0b10100000000000000000000000000000UL;
+    static const uint32_t    ARMING_BUZZ = 0b11111111111111111111111111111100UL; // 3s
+    static const uint32_t      BARO_BUZZ = 0b10101010100000000000000000000000UL;
+    static const uint32_t        EKF_BAD = 0b11110111011010000000000000000000UL;
```

Comment:
Still wrong:
```suggestion
    static const uint32_t        EKF_BAD = 0b11101101010000000000000000000000UL;
```

Reply:
Only picked up the nitpick the first time through.  Fixed this time.